wiki/604838-SubmissionDashboards.md

```diff
@@ -13,6 +13,42 @@ This section provides dashboards to the person who submit NLP Sandbox tools for
 - You can cancel a submission tthat has not started yet.
 - Only submissions that have successfully completed are counted toward your (or your team's) submission quota.
 - If you can't figure out why your submission failed, please [open a ticket].
+- The SubmissionLogs column contains information of the workflow that is ran per submission.  It doens't actually contain logs generated by your submissions.  If your submission is INVALID or REJECTED, you will have to request for logs.
+
+### Submission Status Definitions
+
+We use a the i2b2 dataset to validate the format of the models submitted.
+
+`status` column
+
+{row}
```

Comment:
Replace by standard markdown code whenever possible, for example a table with the columns `Status` and `Description`.

Reply:
I'm confused what the rows would be then.  Are you looking for something like this:

status | description
---|---
[blank] | Waiting to start evaluation
EVALUATION_IN_PROGRESS | Evaluating on site's submission
ACCEPTED | Model created valid prediction file
SCORED | Scored model
INVALID | Model either failed to run on site's data, or prediction file is invalid